File: Osnovne skole/2024/Jedrilica/Jedrilica.py
n = int(input())
lista = []
for i in range(n):
    lista.append(int(input()))

result = 0
for i in range(n - 1):
    diff = max(lista[i], lista[i+1]) - min(lista[i], lista[i+1])
    if diff > 180: diff = 360 - diff
    result += diff

print(result)

File: Osnovne skole/2024/Jedrilica/Jedrilica.cpp
#include<iostream>
#include<vector>
using namespace std;
int main() {
    int N; cin >> N;
    vector<int> arr(N);
    for(int i = 0 ; i < N; i++)
        cin >> arr[i];
    int result = 0;
    for(int i = 0; i < N - 1; i++) {
        int diff = max(arr[i], arr[i+1]) - min(arr[i], arr[i+1]);
        if(diff > 180) diff = 360 - diff;
        result += diff;
    }
    cout << result << endl;
    return 0;
}   